jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-showcase/pom.xml

```diff
@@ -403,6 +407,12 @@
       <artifactId>guava</artifactId>
     </dependency>
 
+    <!-- KIE Workbench Playground Repository -->
+    <dependency>
+      <groupId>org.kie.workbench.playground</groupId>
```

Comment:
@jhrcek it should be covered in here https://github.com/kiegroup/kie-wb-playground/pull/12/files#diff-73d7cad6005591e4b2ffd6e386d7abbbR13